config/manifest-loader.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * A loader that injects code at the bottom of a manifest to automatically execute an optional receiveContentProps function,
+ * passing along the frontmatter of content-files of a matching entry-name.
+ * @param {string} source The manifest.js file content
+ */
+function manifestLoader(source) {
+  return `
+    ${source}
+    (() => {
+      if (module.exports.receiveContentProps) {
+        const applicationSettings = window.settings.applications[module.exports.entryName];
+        module.exports.receiveContentProps(...applicationSettings.contentProps);
+      }
+    })();
+  `;
+}
+
+module.exports = manifestLoader;
```

Comment:
This is the actual loader that Webpack uses to add code into a `manifest.js` that will automatically execute the `receiveContentProps` function with the front matter.